### DISCUSSION TITLE
Failing to enable AIRFLOW__LOGGING__REMOTE_LOGGING

### DISCUSSION THREAD

**Discussion by yashshrivastav22** [apache/airflow#21049] [General]
### Apache Airflow version

2.1.2

### What happened

I have deployed airflow application on kubernetes using simple yamls definition and I am trying to integrate my airflow application with elasticsearch for logging. I have tried to enable the one or more variables in airflow but my airflow webserver is going in crashloopbackoff state while enabling the "remote_logging" variable. This is the only variable (AIRFLOW__LOGGING__REMOTE_LOGGING) where I am trying to enable it and going crashloopbackoff state. FYI I am using airflow 2.1.2

### What you expected to happen

Mentioned variable should be enabled in airflow application for elasticsearch logging.

### Operating System

Debian GNU/Linux 10 (buster)

### Versions of Apache Airflow Providers

airflow 2.1.2

### Deployment

Other Docker-based deployment

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Converted it into discussion. There is no information on reproduction, I am not even sure what your problem is. If you want to continue discussing it with others, then I suggest you to add more details and explain precisely what your problem is. Look at the logs, try to find similar issues first and possibly even discuss it in slack in #troubleshooting but you need to provde more information - maybe someone will be able to help you. Specifically you need to provide stack traces and logs generated by the crash loopbacking pods.

Also I'd suggest you to use the official Helm Chart to install Airflow, rather than try with your own resources https://airflow.apache.org/docs/helm-chart/stable/index.html  - You will have better starting point and be sure that you are not fighting with your own deployment issues.

**Reply by yashshrivastav22** [reply]
This image shows how I am facing the issue in airflow while applying variable in airflow pod. Check this

<img width="960" alt="airflow_remote_logging" src="https://user-images.githubusercontent.com/52623650/150921415-8e7e44ff-9b18-4cb5-a5f6-01dce3507268.PNG">

**Reply by potiuk** [reply]
The error message tels you what to check. What happened when  you followed it? Did you try it? Did you try to search for some blogs and issues? What did your search result with? What kind of problems you experienced when you tried?

**Comment by yashshrivastav22** [comment]
Yes. I tried a lot of time to resolve but not achieved any success in this. Now, I am trying to create custom configuration in airflow application but somehow few of variables are not able to enable.

**Reply by potiuk** [reply]
What did yoiu try? What did not work? What happened when you followed the advice? 

Currently, the only good advice I can give you is ..... folllow the advice that airlfow gives.
